quickstarts/01-hello-world/README.md

```diff
@@ -4,15 +4,25 @@ This quickstart provides a hands-on introduction to Dapr Agents through simple e
 
 ## Prerequisites
 
-- Python 3.10 (recommended)
-- pip package manager
+- Python 3.10+ (recommended)
+- [uv package manager](https://docs.astral.sh/uv/getting-started/installation/)
```

Comment:
I'm ok with this change, but we should double check with yaron and/or roberto that they're fine with updating the "default" pkg manager we say to use... bc i have an open issue to update all quickstarts on this